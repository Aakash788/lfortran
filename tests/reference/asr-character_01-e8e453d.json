{
    "basename": "asr-character_01-e8e453d",
    "cmd": "lfortran --show-asr --no-color {infile} -o {outfile}",
    "infile": "tests/warnings/character_01.f90",
    "infile_hash": "f952599c29229b99c5aca49cbb1e3ee4ff0dbd778b07ce826549ff4d",
    "outfile": null,
    "outfile_hash": null,
    "stdout": "asr-character_01-e8e453d.stdout",
    "stdout_hash": "d0c881a030416d363bc65ebb6aed09ba2e5d71b8e0799eb533203278",
    "stderr": "asr-character_01-e8e453d.stderr",
    "stderr_hash": "c65e821fd013fdfacc225b26b767527afdb4d228d8e60841a7d0349c",
    "returncode": 0
}