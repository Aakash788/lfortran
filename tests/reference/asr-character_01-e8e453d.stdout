(TranslationUnit
    (SymbolTable
        1
        {
            character_01:
                (Program
                    (SymbolTable
                        2
                        {
                            c_null_char:
                                (Variable
                                    2
                                    c_null_char
                                    []
                                    Local
                                    (StringConstant
                                        "ball"
                                        (Character 1 4 ())
                                    )
                                    (StringConstant
                                        "b"
                                        (Character 1 1 ())
                                    )
                                    Parameter
                                    (Character 1 1 ())
                                    ()
                                    Source
                                    Public
                                    Required
                                    .false.
                                )
                        })
                    character_01
                    []
                    []
                )
        })
    []
)
